<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmSliderBar</class>
 <widget class="QWidget" name="frmSliderBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="SliderBar" name="widgetLeft" native="true">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>7</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="SliderBar" name="widget1" native="true"/>
      </item>
      <item>
       <widget class="SliderBar" name="widget2" native="true"/>
      </item>
      <item>
       <widget class="SliderBar" name="widget3" native="true"/>
      </item>
      <item>
       <widget class="SliderBar" name="widget4" native="true"/>
      </item>
      <item>
       <widget class="SliderBar" name="widget5" native="true"/>
      </item>
      <item>
       <widget class="SliderBar" name="widget6" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SliderBar</class>
   <extends>QWidget</extends>
   <header>sliderbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
